core/src/main/java/io/questdb/cairo/map/FastMapRecord.java

```diff
@@ -342,35 +351,28 @@ protected MapRecord clone() {
         return new FastMapRecord(valueOffsets, split, keyDataOffset, keyBlockOffset, csA, csB, bs, long256A, long256B);
     }
 
+    @NotNull
+    private Long256 getLong256Generic(Long256Impl[] array, int columnIndex) {
+        long address = addressOfColumn(columnIndex);
+        Long256Impl long256 = array[columnIndex];
+        long256.setAll(
+                Unsafe.getUnsafe().getLong(address),
+                Unsafe.getUnsafe().getLong(address + Long.BYTES),
+                Unsafe.getUnsafe().getLong(address + Long.BYTES * 2),
+                Unsafe.getUnsafe().getLong(address + Long.BYTES * 3)
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

The result of `Integer` multiplication is cast into a larger type. If you think the result of the multiplication could exceed the size of an `Integer`, consider casting the arguments of the multiplication to avoid arithmetic overflows. For more information, see [CWE-190](https://cwe.mitre.org/data/definitions/190.html).